src/tests/unit/tests/common/store-proxy.test.ts

```diff
@@ -78,17 +77,16 @@ describe('StoreProxyTest', () => {
             tabId: 1,
             storeType: StoreType.TabContextStore,
             storeId: 'TestStore',
-            isStoreUpdateMessage: true,
             payload: 'store state',
-        } as StoreUpdateMessage<string>);
+        } as unknown as StoreUpdateMessage<string>);
```

Comment:
I think these last 2 test cases should probably just be removed, they're the distributor's responsibility now.